src/ResourceManager/EventHub/Commands.EventHub/Utilities/EventHubsClient.cs

```diff
@@ -300,6 +301,54 @@ public ListKeysAttributes SetRegenerateKeys(string resourceGroupName, string nam
 
         #endregion
 
+        #region DRConfiguration
+        public EventHubDRConfigurationAttributes GetEventHubDRConfiguration(string resourceGroupName, string namespaceName, string alias)
+        {
+            var response = Client.DisasterRecoveryConfigs.Get(resourceGroupName, namespaceName, alias);
+            return new EventHubDRConfigurationAttributes(response);
+        }
+
+        public IEnumerable<EventHubDRConfigurationAttributes> ListAllEventHubDRConfiguration(string resourceGroupName, string namespaceName)
+        {
+            var response = Client.DisasterRecoveryConfigs.List(resourceGroupName, namespaceName);
+            var resourceList = response.Select(resource => new EventHubDRConfigurationAttributes(resource));
+            return resourceList;
+        }
+
+        public EventHubDRConfigurationAttributes CreateEventHubDRConfiguration(string resourceGroupName, string namespaceName, string alias, EventHubDRConfigurationAttributes parameter)
+        {
+            var Parameter1 = new Management.EventHub.Models.ArmDisasterRecovery();
+
+            if (!string.IsNullOrEmpty(parameter.PartnerNamespace))
+                Parameter1.PartnerNamespace = parameter.PartnerNamespace;
+
+            var response = Client.DisasterRecoveryConfigs.CreateOrUpdate(resourceGroupName, namespaceName, alias, Parameter1);
+            Thread.Sleep(TimeSpan.FromSeconds(5));
```

Comment:
@v-Ajnava why are you calling `Thread.Sleep`?

Reply:
Removed, these were used to avoid getting the cached object.